FAQ.rst

```diff
@@ -0,0 +1,269 @@
+      23.                 return view_func(request, *args, **kwargs)
+
+    File "C:\Users\hansong.li\Documents\GitHub\equipCal\calbase\views.py" in post_update
+      129.     return render(request, 'calbase/default_edit.html', context)
+
+    File "C:\Users\hansong.li\AppData\Local\Programs\Python\Python35-32\lib\site-packages\django\shortcuts.py" in render
+      30.     content = loader.render_to_string(template_name, context, request, using=using)
+
+    File "C:\Users\hansong.li\AppData\Local\Programs\Python\Python35-32\lib\site-packages\django\template\loader.py" in render_to_string
+      68.     return template.render(context, request)
+
+    File "C:\Users\hansong.li\AppData\Local\Programs\Python\Python35-32\lib\site-packages\django\template\backends\django.py" in render
+      66.             return self.template.render(context)
+
+    File "C:\Users\hansong.li\AppData\Local\Programs\Python\Python35-32\lib\site-packages\django\template\base.py" in render
+      208.                     return self._render(context)
+
+    File "C:\Users\hansong.li\AppData\Local\Programs\Python\Python35-32\lib\site-packages\django\template\base.py" in _render
+      199.         return self.nodelist.render(context)
+
+    File "C:\Users\hansong.li\AppData\Local\Programs\Python\Python35-32\lib\site-packages\django\template\base.py" in render
+      994.                 bit = node.render_annotated(context)
+
+    File "C:\Users\hansong.li\AppData\Local\Programs\Python\Python35-32\lib\site-packages\django\template\base.py" in render_annotated
+      961.             return self.render(context)
+
+    File "C:\Users\hansong.li\AppData\Local\Programs\Python\Python35-32\lib\site-packages\django\template\loader_tags.py" in render
+      174.         return compiled_parent._render(context)
+
+    File "C:\Users\hansong.li\AppData\Local\Programs\Python\Python35-32\lib\site-packages\django\template\base.py" in _render
+      199.         return self.nodelist.render(context)
+
+    File "C:\Users\hansong.li\AppData\Local\Programs\Python\Python35-32\lib\site-packages\django\template\base.py" in render
+      994.                 bit = node.render_annotated(context)
+
+    File "C:\Users\hansong.li\AppData\Local\Programs\Python\Python35-32\lib\site-packages\django\template\base.py" in render_annotated
+      961.             return self.render(context)
+
+    File "C:\Users\hansong.li\AppData\Local\Programs\Python\Python35-32\lib\site-packages\django\template\loader_tags.py" in render
+      70.                 result = block.nodelist.render(context)
+
+    File "C:\Users\hansong.li\AppData\Local\Programs\Python\Python35-32\lib\site-packages\django\template\base.py" in render
+      994.                 bit = node.render_annotated(context)
+
+    File "C:\Users\hansong.li\AppData\Local\Programs\Python\Python35-32\lib\site-packages\django\template\base.py" in render_annotated
```

Comment:
Can you remove this everywhere please ?

    C:\Users\hansong.li\AppData\Local\Programs\Python\Python35-32\lib\

It's not needed